samples/sample_monthly_update.md

```diff
@@ -0,0 +1,70 @@
+# Sample Update
+
+We ask our companies to send us regular updates so that we can stay up to date with the company, and help out as necessary. To help us help you, here is a sample of what we need to see.
+
+We ask that you spend at most 15-30 minutes filling out this information, as you will want systems in place to easily track this data.
+
+## Outline
+
+* **Our Victories**
+* **Our Struggles**
+* **Key Metrics**
+* **How You Can Help**
+* **Other Notes**
+
+---
+
+## Teacher Tools Inc. - May Update
```

Comment:
My goal was to make this as believable as possible but it feels a little sterile. Maybe that's okay?